integration-tests/contract-originate-contract-with-sapling.spec.ts

```diff
@@ -43,5 +61,132 @@ CONFIGS().forEach(({ lib, rpc, setup }) => {
 
       done();
     });
+
+    test('Originates a Sapling Double contract', async (done) => {
+      const op = await Tezos.contract.originate({
+        code: saplingContractDouble,
+        init: `(Pair {} {})`,
+      });
+      await op.confirmation();
+      expect(op.hash).toBeDefined();
+      expect(op.includedInBlock).toBeLessThan(Number.POSITIVE_INFINITY);
+      const contract = await op.contract();
+
+      const saplingDiffById = await rpcClient.getSaplingDiffById('168');
```

Comment:
The sapling state id 168 is not related to the current contract, should this be removed from the test?

Reply:
removed.